### PR TITLE
Add ruleset: UAB.edu

### PR DESCRIPTION
**Pull request by dnoetzel**
Fixes issue #16915.

### CODE REVIEW

**Comment by J0WI** [comment]
It would be great if you could add as many targets as possible: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

File: src/chrome/content/rules/UAB.edu.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Nonfunctional hosts in *.uab.edu:
+		- mail.uab.edu
+		- web.uab.edu
+-->
+<ruleset name="UAB.edu">
```
Comment (J0WI) [inline src/chrome/content/rules/UAB.edu.xml]: Please rename to `UAB.edu (partial)`

**Comment by dnoetzel** [comment]
@J0WI 
> It would be great if you could add as many targets as possible: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

Thanks for pointing me in the right direction. `Sublist3r` reports 1,467 domains associated with `uab.edu`, 110 of which were reachable only via HTTP and 269 via HTTPS, but  of those 269 subdomains, 98 of them now just redirect to some location on `www.uab.edu` (e.g. `http://sa.psychiatry.uab.edu` ultimately redirects to `https://www.uab.edu/medicine/substanceabuse/`). While I'm sure that wildcards could cut down on the number of entries somewhat, listing out hundreds of targets seems like it would bloat the ruleset. How would you recommend I go about listing these targets?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**